README.md

```diff
@@ -24,64 +24,355 @@
+    }
+}
+```
+## returns this if no user is found with that email address
+
+```
+{
+    "data": []
+}
+```
+
+
+## Post or Create a new user to the backend database
+post 'api/v1/users'
 
-<img src="./app/assets/images/friends_requests_pending_endpoint.jpg" alt="This is what we expect to return when we query for friends of a user with status pending"  />
+pass in name, google_id, and email in headers
+
+```
+{
+    "message": "User successfully created"
+}
+```
+
+
+
+## Posts [And by Posts we mean posts that a user makes on their dashboard]
+## Get all posts of a user (history)
+
+get '/api/v1/users/history' & pass in user through headers
+or
+get '/api/v2/users/history?user=<google_id>'
+
+ ``` 
+{ “data” : 
+	[
+		{ 
+			“id”: 1 ,
+		“type": "post",
+		"attributes": {
+				"emotion": "pensive"
+				"post_status": "public"
+				"description": "This was what I typed when I made this post."	 
+				"tone" : "anger"
+				"created_at": "2022-11-01 17:02:19.036593 -0600"	 
+			}
+		},
+		{ 
+			“id”: 2 ,
+		“type": "post",
+		"attributes": {
+				"emotion": "pensive"
+				"post_status": "public"
+				"description": "This was what I typed when I made this post."	 
+				"tone" : "anger"
+				"created_at": "2022-11-01 17:02:19.036593 -0600"	 
+			}
+		},
+		{ 
+			“id”: 3 ,
+		“type": "post",
+		"attributes": {
+				"emotion": "pensive"
+				"post_status": "public"
+				"description": "This was what I typed when I made this post."	 
+				"tone" : "anger"
+				"created_at": "2022-11-01 17:02:19.036593 -0600"	 
+			}	
+		}
+	]
+}
+  ``` 
+
+## Get the last post of a user (recent post)
+
+get 'api/v2/users/last?user=<google_id>'
+
+ ``` 
+{ “data” : 
+		{ 
+			“id”: 3 ,
+		“type": "post",
+		"attributes": {
+				"emotion": "pensive"
+				"post_status": "public"
+				"description": "This was what I typed when I made this post."	 
+				"tone" : "anger"
+				"created_at": "2022-11-01 17:02:19.036593 -0600"	 
+			}	
+		}
+}
+  ``` 
 
 
-## Get all friends of a user with a status of accepted
-get 'api/v1/friends?request_status=accepted'
 
-<img src="./app/assets/images/friends_requests_accepted_endpoint.jpg" alt="This is what we expect to return when we query for friends of a user with status pending"  />
+## create a new post
+post '/api/v2/posts?user=<google_id>&emotion=<emotion_term>&description=<description>&post_status=<personal or shared>'
+
+# if the post was successfully created
+```
+{
+  “data” : 
+  {
+    “id”: 808 ,
+   “type": "post",
+   "attributes": {
+     "emotion": "thrilled",
+     "post_status": "personal",
+     "description": "Today I got a job offer for the company that I really loved, and have admired for a long time.",
+     "tone": "joy",
+     "created_at": "2022-11-05T23:05:26.971Z"
+    }
+  }
+}
+```
+# if the post was not successfully created
+```
+{
+    "data": []
+}
+```
+## update an existing post
+patch '/api/v2/posts?user=<google_id>&emotion=<emotion_term>&description=<description>&post_status=<personal or shared>'
```

Comment:
the PATCH request need the post id in the URI so it will be:
 - `/api/v2/posts/:post_id?emotion=<emotion_term>&description=<description>&post_status=<personal or shared>`
 with the post id, we will have user id already so google_id won't be needed.

Reply:
corrected